cr17/nodes/cr17/obstacle_mapper.py

```diff
@@ -1,15 +1,38 @@
-#!/usr/bin/env python
+#!/usr/bin/env python22
```

Comment:
yea, I was wondering about that, but since the node runs fine I was going to leave it